.jshintignore

```diff
@@ -0,0 +1,33 @@
+# Logs
```

Comment:
My bad, I was using it for local changes not in this PR yet. It isn't needed (yet?)